### DISCUSSION TITLE
Using LibSumo with Eclipse MOSAIC

### DISCUSSION THREAD

**Discussion by alizz-99** [eclipse/mosaic#366] [General]
Hi, I am trying to capture data that SUMO creates in the simulation from an external python script. Reading the documentation I understand that I have to do it with LibSumo but when I modify the file <mosaic>/etc/runtime.json, I get the following error:

`--------------------------------------------------------------------------------
 Stopping simulation due to a critical error:
	- IllegalArgumentException: Invalid Number Format (double) 1.00
	- Root Cause: IllegalArgumentException: Invalid Number Format (double) 1.00
 Please see the log files for details.
	- Log-Directory: logs/log-20231110-082028-Barnim
--------------------------------------------------------------------------------
 MOSAIC will now shut down.
--------------------------------------------------------------------------------`

I hope you can help me, or give me new ideas on how to do it. Thank you very much!

**Comment by schwepmo** [comment]
Hello, sorry for getting back to you so late. If you just want to capture data from SUMO given a SUMO scenario via python then there is no need to use MOSAIC. Otherwise, depending on what data you want to try to capture the [FileOutput](https://eclipse.dev/mosaic/docs/visualization/filevis/) maybe useful to you.

I don't think it is possible for SUMO to connect to MOSAIC and your python script at the same time.

As for you error message, this could have multiple reasons, to further identify the issue could you attach your `MOSAIC.log` and the `traffic.log` generated when running MOSAIC?
